test/parallel/test-fs-watchfile.js

```diff
@@ -37,24 +36,13 @@ const expectedStatObject = new fs.Stats(
     Date.UTC(1970, 0, 1, 0, 0, 0)             // birthtime
 );
 
-function removeTestFile() {
-  try {
-    fs.unlinkSync(enoentFile);
-  } catch (ex) {
-    if (ex.code !== 'ENOENT') {
-      throw ex;
-    }
-  }
-}
-
-// Make sure that the file does not exist, when the test starts
-removeTestFile();
+common.refreshTmpDir();
 
 // If the file initially didn't exist, and gets created at a later point of
 // time, the callback should be invoked again with proper values in stat object
 var fileExists = false;
 
-fs.watchFile(enoentFile, common.mustCall(function(curr, prev) {
+fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
```

Comment:
I suppose the whole "what happens if you set `interval` to `0`" thing should be covered in the docs. /cc @nodejs/documentation